testing-modules/junit-5/src/test/java/com/baeldung/execution/time/SampleExecutionTimeUnitTest.java

```diff
@@ -0,0 +1,46 @@
+package com.baeldung.execution.time;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class SampleExecutionTimeUnitTest {
+
+    @Test
+    public void someUnitTest() {
+
+        assertTrue(this::doSomething);
+    }
+
+    @Test
+    public void someIntegrationTest() {
+
+        try {
+            //simulate an operation that may take 5 seconds
+            Thread.sleep(5000);
+        } catch (InterruptedException e) {
+            fail();
+        }
+
+        assertTrue(this::doSomething);
+    }
+
+    @Test
+    public void someEndToEndTest() {
+
+        try {
+            //simulate an operation that may take 10 seconds
+            Thread.sleep(10000);
+        } catch (InterruptedException e) {
+            fail();
+        }
```

Comment:
This is distracting - the try/catch, which are just caused by the use of `Thread.sleep` Just make your test methods `throws Exception`